central/convert/internaltov2storage/compliance_operator_scan.go

```diff
@@ -0,0 +1,43 @@
+package internaltov2storage
+
+import (
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+const (
+	ocpComplianceLabelsKey = "compliance.openshift.io/"
```

Comment:
Is this available anywhere in the compliance code that we can reference it instead of declaring it?

Reply:
Or we define this twice, can this go into pkg?

Reply:
Actually 3 times as it was declared in the sensor side dealing with results.  Good news is there are constants in compliance code